packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -124,8 +113,16 @@ export const SELL_SOURCE_FILTER_BY_CHAIN_ID = valueByChainId<SourceFilters>(
             ERC20BridgeSource.JulSwap,
             ERC20BridgeSource.LiquidityProvider,
         ]),
+        [ChainId.Polygon]: new SourceFilters([
```

Comment:
I think DODO v1 is also deployed on Polygon https://dodoex.github.io/docs/docs/deployedInfoMatic#dodo-v1

Reply:
Integrated and verified working 👍